### ISSUE TITLE
simkl.com

### ISSUE THREAD

**Issue by AdamWr** [AdguardTeam/CoreLibs#1027]
@adguard-bot commented on [Mon Jul 15 2019](https://github.com/AdguardTeam/AdguardFilters/issues/36856)

### Issue URL (Incorrect Blocking)
[https://simkl.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsimkl.com%2F)
### Comment
Username: @xKeiro
> If I try to navigate to anywhere on the website with Adguard enabled the site just blurs and nothing loads in until I hit F5  to refresh the page (Example: on the example image I'm trying to click on the "Anime" button). The problem still persists if I disable Adblocking  for the website in the Adguard Assistant the website only starts functioning properly if I disable adguard altogether in the windows app.
> --
> The information below is automatically generated. Do not delete it.
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  ![Screenshot 1](https://reports-img.adguard.com/uMPMz2W.jpg)
</details>
<details>
  <summary>Screenshot 2</summary>

  ![Screenshot 2](https://reports-img.adguard.com/D86daSA.jpg)
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.1.2817.0
Browser: | Chromium Edge
AdGuard driver: | WFP
Filters: | Filter unblocking search ads and self-promotion,<br>EasyList Italy,<br>Icelandic ABP List,<br>AdGuard Annoyances,<br>AdGuard Base,<br>Dandelion Sprout's Nordic Filters,<br>AdGuard Tracking Protection

**Comment by AdamWr** [comment]
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/61205442-58948100-a6f0-11e9-8ce9-281d7988f884.png)

</details>

Added as a temporary solution `@@||simkl.com^$elemhide,jsinject,extension`, so please disable this rule before checking it.

**Comment by ameshkov** [comment]
Merged into https://github.com/AdguardTeam/CoreLibs/issues/1221
